file-formats/cfdf/examples/yy1_aff_example.cfdf.json

```diff
@@ -0,0 +1,202 @@
+{
+  "formatVersion": "1 ",
+  "header": {
+    "description": "YY1_AFF_Example",
+    "originalLanguage": "en"
+  },
+  "externalFieldName": "YY1_AFF_Example",
```

Comment:
The external field name can have different values than the object name, right?

Reply:
The external name is case sensitive as the capitalization is relevant in some artefacts (like OData property names are case sensitive). In customer systems, the transport object name and external name are equals except of the capitalization. In internal SAP systems, custom fields usually are created as Z-objects, but the field names (used in extensions of SAP objects) must not lie in the Z namespace. Therefore, the object name is often something like ZZ1_* and the external field name would be something like ZX1_*.

Reply:
OK, I just wanted to make sure that we do not repeat the object name which is represented in the filename.